### DISCUSSION TITLE
Add documentation for Kit usage in AMP pages

### DISCUSSION THREAD

**Discussion by the-architect** [FortAwesome/Font-Awesome#20952] [Feature Requests]
**Describe the bug**
I have no idea how to use FA kit scripts in AMP pages to display icons.

**Expected behavior**
Explanation in README or a guide "How to use FA in AMP pages"

**Bug report checklist**
- [x] I have filled out as much of the above information as I can
- [x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by lesleslie** [comment]
I have a request for this as well. It would most likely require having https://kit.fontawesome.com/ whitelisted for .js from the AMP team. I'm going to post this request to the AMP Slack and see if a feature request needs to be filed for this.

**Comment by jodydonetti** [comment]
any news on this?

**Comment by lesleslie** [comment]
Not that I've heard. I haven't looked into it yet, but it may be possible to implement using amp-script now that that's been released.

**Comment by esng25** [comment]
Font Awesome CDN still exists ([https://fontawesome.com/account/cdn](https://fontawesome.com/account/cdn)) but is being sunsetted from further support as the Font Awesome team is pushing the use of kit.fontawesome.com. Please consider whitelisting this sub-domain so we can access the latest icons and features from Font Awesome.
